components/infra-proxy-service/migrations/migrations.go

```diff
@@ -365,3 +372,211 @@ func SetStagedUserForConfirmPreview(users []*request.User) []pipeline_model.User
 	}
 	return usersData
 }
+
+// CreateBackup Creates sample knife ec back up file
+func (s *MigrationServer) CreateBackup(ctx context.Context, req *request.CreateBackupRequest) (*response.CreateBackupResponse, error) {
+
+	backupPath := "/src/backup/"
+
+	// Create org directory
+	orgPath := path.Join(backupPath, "organizations/", req.OrgId, "groups")
+	if _, err := os.Stat(orgPath); os.IsNotExist(err) {
+		err := os.MkdirAll(orgPath, os.ModePerm)
+		if err != nil {
+			log.Errorf("Unable to create directory %s", err.Error())
+			return nil, err
+		}
+	}
+
+	orgJsonPath := path.Join(backupPath, "organizations", req.OrgId, "org.json")
+	keyDumpJsonPath := path.Join(backupPath, "key_dump.json")
+	membersJsonPath := path.Join(backupPath, "organizations", req.OrgId, "members.json")
+	adminsJsonPath := path.Join(backupPath, "organizations", req.OrgId, "groups", "admins.json")
+
+	// Create data for org file org.json
+	orgJsonObj := pipeline.OrgJson{}
+	orgJsonObj.Name = req.OrgId
+	orgJsonObj.FullName = "f_" + req.OrgId
+
+	err := writeOrgFile(orgJsonObj, orgJsonPath)
+	if err != nil {
+		log.Errorf("Failed to write org file org.json ", err)
+		return nil, err
+	}
+
+	// Create an object for key_dump.json
+	serverUsers := []pipeline.KeyDump{}
+
+	if _, err := os.Stat(keyDumpJsonPath); err == nil {
+		data, err := ioutil.ReadFile(keyDumpJsonPath)
+		if err != nil {
+			return nil, err
+		}
+		err = json.Unmarshal(data, &serverUsers)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Create an object for members.json
+	members := []pipeline.MembersJson{}
+
+	// Create an object for admins.json
+	admins := pipeline.AdminsJson{}
+	admins.Name = "admins"
+	admins.Users = append(admins.Users, "pivotal")
+
+	// Random generator to set the values for ExternalAuthenticationUID and admins randomly
+	rand.Seed(time.Now().Unix())
+	users := []string{pipeline.Local, pipeline.LDAP}
+	isAdmins := []bool{true, false}
+
+	for i := 0; i < int(req.NumberOfUsers); i++ {
+
+		// Create an object for key_dump.json
+		keyDump := pipeline.KeyDump{}
+		nowTime := time.Now().Format("20060102150405")
+		keyDump.ID = "user_" + uuid.Must(uuid.NewV4()).String()
+
+		keyDump.Username = keyDump.ID
```

Comment:
The username should be sequential name:
```suggestion
		keyDump.Username = `user_${i}`
```